components/brave_wallet/browser/internal/orchard_storage/orchard_types.h

```diff
@@ -0,0 +1,132 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_ORCHARD_STORAGE_ORCHARD_TYPES_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_ORCHARD_STORAGE_ORCHARD_TYPES_H_
+
+#include <utility>
+#include <vector>
+
+#include "base/memory/raw_ref.h"
+#include "brave/components/brave_wallet/common/zcash_utils.h"
+
+namespace brave_wallet {
```

Comment:
all of these types appear to be associated with orchard_storage, should these either move to `orchard_storage.h` or rename this to `orchard_storage_types.h`?

Reply:
or are these not directly associated with orchard_storage and it just happens to be the only code that uses them right now?